package.json

```
@@ -1,14 +1,14 @@
 {
   "name": "react-native-signature-capture",
```

Make sure you rename this to `@localz/...`

```suggestion
  "name": "@localz/react-native-signature-capture",
```